docs/usage/types/extra_types/extra_types.md

```diff
@@ -8,5 +8,6 @@ The following types are supported by [Pydantic Extra Types]:
 * [Payment Card Numbers](payment_cards.md) &mdash; a type that allows you to store payment card numbers in your model.
 * [Phone Numbers](phone_numbers.md) &mdash; a type that allows you to store phone numbers in your model.
 * [Routing Numbers](routing_numbers.md) &mdash; a type that allows you to store ABA transit routing numbers in your model.
+* [Mac Address](mac_address.md) &mdash; a type that allows you to store Mac Address in your model.
```

Comment:
```suggestion
* [MAC Address](mac_address.md) &mdash; a type that allows you to store MAC addresses in your model.
```

Reply:
You may want to update the docs for your extra types PR with this usage. Feel free to tag me for review.